core-services/image-mirroring/openshift/mapping_origin_4_10

```diff
@@ -159,6 +159,8 @@ registry.ci.openshift.org/origin/4.10:loki quay.io/openshift/origin-loki:4.10 qu
 registry.ci.openshift.org/origin/4.10:machine-api-operator quay.io/openshift/origin-machine-api-operator:4.10 quay.io/openshift/origin-machine-api-operator:4.10.0 quay.io/openshift/origin-machine-api-operator:latest
 registry.ci.openshift.org/origin/4.10:manila-csi-driver-operator-test quay.io/openshift/origin-manila-csi-driver-operator-test:4.10 quay.io/openshift/origin-manila-csi-driver-operator-test:4.10.0 quay.io/openshift/origin-manila-csi-driver-operator-test:latest
 registry.ci.openshift.org/origin/4.10:mdns-publisher quay.io/openshift/origin-mdns-publisher:4.10 quay.io/openshift/origin-mdns-publisher:4.10.0 quay.io/openshift/origin-mdns-publisher:latest
+registry.ci.openshift.org/origin/4.10:meta-cluster-api-operator quay.io/openshift/origin-meta-cluster-api-operator:4.10 quay.io/openshift/origin-meta-cluster-api-operator:4.10.0 quay.io/openshift/origin-meta-cluster-api-operator:latest
```

Comment:
Naming review:

Ok, why is this called "meta"?  From a naming perpective I don't understand what "meta" means, so if you can articulate that it'll help me review.